snap/info_snap_yaml.go

```diff
@@ -369,6 +371,10 @@ func setAppsFromSnapYaml(y snapYaml, snap *Info) error {
 				Timer: yApp.Timer,
 			}
 		}
+		// collect all common IDs
+		if app.CommonId != "" {
+			snap.CommonIds = append(snap.CommonIds, app.CommonId)
```

Comment:
do we know if the same common-id can be repeated ?  should we list it only once?

Reply:
result of discussion is that it should not be repeated and we need to validate for that